ships/botany_bay.json

```diff
@@ -0,0 +1,3008 @@
+            "materials": [],
+            "others": [],
+            "resources": []
+          },
+          "img_url": "./assets/component_mining_laser.png",
+          "quantity": 1,
+          "repair_cost": []
+        }
+      ],
+      "levels": [
+        {
+          "ability_bonus": 265,
+          "additional_officer_slot": false,
+          "level": 1,
+          "ship_xp": 0
+        },
+        {
+          "ability_bonus": 268,
+          "additional_officer_slot": false,
+          "level": 2,
+          "ship_xp": 800
+        },
+        {
+          "ability_bonus": 272,
+          "additional_officer_slot": false,
+          "level": 3,
+          "ship_xp": 1000
+        },
+        {
+          "ability_bonus": 275,
+          "additional_officer_slot": false,
+          "level": 4,
+          "ship_xp": 1400
+        },
+        {
+          "ability_bonus": 278,
+          "additional_officer_slot": false,
+          "level": 5,
+          "ship_xp": 1700
+        }
+      ],
+      "repair_time": 1590,
+      "tier": 1
+    },
+    {
+      "build_time": 96000,
+      "components": [
+        {
+          "name": "Photon Torpedo",
+          "additional_info": {
+            "armor_pierce": 131,
+            "crit_chance": 0.1,
+            "crit_modifier": 1.5,
+            "damage_max": 492,
+            "damage_min": 402,
+            "firing_pattern": {
+              "pattern": [
+                1,
+                0,
+                1,
+                0,
+                1,
+                0,
+                1,
+                0,
+                1
+              ],
+              "weapon_type": "kinetic"
+            },
+            "shield_pierce": 131
+          },
+          "build_cost": {
+            "materials": [
+              {
+                "type": "gas",
+                "grade": 2,
+                "rarity": "common",
+                "value": 90
+              },
+              {
+                "type": "ore",
+                "grade": 2,
+                "rarity": "common",
+                "value": 240
+              }
+            ],
+            "others": [
+              {
+                "type": "2\u2605\u00a0Survey\u00a0Parts",
+                "value": 60
+              }
+            ],
+            "resources": [
+              {
+                "type": "tritanium",
+                "value": 6750
+              },
+              {
+                "type": "dilithium",
+                "value": 120
+              }
+            ]
+          },
+          "img_url": "./assets/component_weapon_kinetic.png",
+          "quantity": 1,
+          "repair_cost": [
+            {
+              "type": "tritanium",
+              "value": 47
+            },
+            {
+              "type": "dilithium",
+              "value": 4
+            }
+          ]
+        },
+        {
+          "name": "Cargo Bay",
+          "additional_info": {
+            "cargo_info": {
```

Comment:
This is unrelated to the data or the structure, but I'm having trouble interpreting what this says :')